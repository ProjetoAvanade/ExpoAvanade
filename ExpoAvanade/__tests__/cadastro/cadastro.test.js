import React from 'react';
import renderer from 'react-test-renderer';

import Cadastro from '../../src/screens/cadastro';

it('renders correctly', () => {
  const tree = renderer.create(<Cadastro />).toJSON();
  expect(tree).toMatchSnapshot();
});