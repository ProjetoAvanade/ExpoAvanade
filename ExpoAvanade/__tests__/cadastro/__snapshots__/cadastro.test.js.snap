// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "backgroundColor": "#ffffff",
      "flex": 1,
    }
  }
>
  <View
    style={
      Object {
        "height": "4.3%",
      }
    }
  />
  <View
    style={
      Object {
        "backgroundColor": "#F3BC2C",
        "height": "7.6%",
        "justifyContent": "center",
        "width": "100%",
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "flexDirection": "row",
          "justifyContent": "space-between",
          "marginLeft": "4.7%",
          "width": "59%",
        }
      }
    >
      <View
        accessible={true}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "opacity": 1,
          }
        }
      >
        <Image
          source={1}
          style={
            Object {
              "height": 21.56,
              "width": 25,
            }
          }
        />
      </View>
      <Text
        style={
          Object {
            "fontFamily": "Poppins_700Bold",
            "fontSize": 25,
          }
        }
      >
        Cadastro
      </Text>
    </View>
  </View>
  <View
    style={
      Object {
        "backgroundColor": "#FFFFFF",
        "height": "100%",
        "width": "100%",
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
        }
      }
    >
      <TextInput
        allowFontScaling={true}
        onChangeText={[Function]}
        placeholder="Nome Completo"
        placeholderTextColor="#000000"
        rejectResponderTermination={true}
        style={
          Object {
            "backgroundColor": "#FFFFFF",
            "borderColor": "#F3BC2C",
            "borderRadius": 5,
            "borderWidth": 2,
            "height": "4.5%",
            "marginTop": "8%",
            "paddingLeft": 20,
            "width": "70%",
          }
        }
        underlineColorAndroid="transparent"
        value="algm"
      />
      <TextInput
        allowFontScaling={true}
        onChangeText={[Function]}
        placeholder="CPF"
        placeholderTextColor="#000000"
        rejectResponderTermination={true}
        style={
          Object {
            "backgroundColor": "#FFFFFF",
            "borderColor": "#F3BC2C",
            "borderRadius": 5,
            "borderWidth": 2,
            "height": "4.5%",
            "marginTop": "8%",
            "paddingLeft": 20,
            "width": "70%",
          }
        }
        underlineColorAndroid="transparent"
        value="111192928"
      />
      <TextInput
        allowFontScaling={true}
        onChangeText={[Function]}
        placeholder="Endereço de e-mail"
        placeholderTextColor="#000000"
        rejectResponderTermination={true}
        style={
          Object {
            "backgroundColor": "#FFFFFF",
            "borderColor": "#F3BC2C",
            "borderRadius": 5,
            "borderWidth": 2,
            "height": "4.5%",
            "marginTop": "8%",
            "paddingLeft": 20,
            "width": "70%",
          }
        }
        underlineColorAndroid="transparent"
        value="aglm@gmail.com"
      />
      <TextInput
        allowFontScaling={true}
        keyboardType="default"
        onChangeText={[Function]}
        passwordRules={true}
        placeholder="Senha"
        placeholderTextColor="#000000"
        rejectResponderTermination={true}
        secureTextEntry={true}
        style={
          Object {
            "backgroundColor": "#FFFFFF",
            "borderColor": "#F3BC2C",
            "borderRadius": 5,
            "borderWidth": 2,
            "height": "4.5%",
            "marginTop": "8%",
            "paddingLeft": 20,
            "width": "70%",
          }
        }
        underlineColorAndroid="transparent"
        value="algu7171"
      />
      <TextInput
        allowFontScaling={true}
        keyboardType="default"
        onChangeText={[Function]}
        passwordRules={true}
        placeholder="Confirmar Senha"
        placeholderTextColor="#000000"
        rejectResponderTermination={true}
        secureTextEntry={true}
        style={
          Object {
            "backgroundColor": "#FFFFFF",
            "borderColor": "#F3BC2C",
            "borderRadius": 5,
            "borderWidth": 2,
            "height": "4.5%",
            "marginTop": "8%",
            "paddingLeft": 20,
            "width": "70%",
          }
        }
        underlineColorAndroid="transparent"
        value="algu7171"
      />
      <Text
        style={
          Object {
            "color": "#00ff00",
            "fontFamily": "ABeeZee_400Regular",
            "fontSize": 14,
          }
        }
      >
        Senha iguais!
      </Text>
      <TextInput
        allowFontScaling={true}
        onChangeText={[Function]}
        placeholder="AAAA/MM/DD"
        placeholderTextColor="#000000"
        rejectResponderTermination={true}
        style={
          Object {
            "backgroundColor": "#FFFFFF",
            "borderColor": "#F3BC2C",
            "borderRadius": 5,
            "borderWidth": 2,
            "height": "4.5%",
            "marginTop": "8%",
            "paddingLeft": 20,
            "width": "70%",
          }
        }
        underlineColorAndroid="transparent"
        value="11/11/2011"
      />
      <View
        accessible={true}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#F3BC2C",
            "borderRadius": 5,
            "flexDirection": "row",
            "height": "8.5%",
            "justifyContent": "space-between",
            "marginTop": "8%",
            "opacity": 1,
            "paddingLeft": 20,
            "paddingRight": 20,
            "width": "70%",
          }
        }
      >
        <Text>
          Foto
        </Text>
        <Image
          source={1}
          style={
            Object {
              "height": 21.56,
              "width": 25,
            }
          }
        />
      </View>
      <View
        style={
          Object {
            "padding": 10,
          }
        }
      />
      <View
        accessible={true}
        focusable={true}
        onClick={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Object {
            "alignItems": "center",
            "backgroundColor": "#F3BC2C",
            "borderRadius": 5,
            "height": "8%",
            "justifyContent": "center",
            "marginTop": "8%",
            "opacity": 1,
            "width": "40%",
          }
        }
      >
        <Text
          style={
            Object {
              "color": "#000000",
              "fontFamily": "Poppins_700Bold",
              "fontSize": 25,
            }
          }
        >
          Cadastrar
        </Text>
      </View>
      <Text
        style={
          Object {
            "color": "#797979",
            "fontFamily": "ABeeZee_400Regular",
            "fontSize": 14,
            "marginTop": "8%",
          }
        }
      >
        Você será reenchaminhado para a tela de login
      </Text>
    </View>
  </View>
</View>
`;
